src/hooks/useGetPaidCourses.js

```diff
@@ -11,7 +11,6 @@ const useGetPaidCourses = (courseMetaData) => {
       );
       return apiRes.data;
     } catch (err) {
-      console.error(err);
```

Comment:
I noticed you've removed all `consol.error()`s. this way no information will passed to debug the error

Reply:
Seems like the code catches the error then returns null. @CodeWithEmad  Are you worried about leaking api data?

Reply:
yup. other than rising warnings on `npm run lint`, I have concerns about API data leakage.